src/language/nodes.py

```diff
@@ -220,13 +220,23 @@ def get_add_methods(self):
                           */
                          void add{self.class_name}({self.class_name} *n) {{
                              {self.varname}.emplace_back(n);
+                             // set parents
+                             // this check could be superfluous, may we add nullptr as children?
```

Comment:
```suggestion
                             // this check could be superfluous, may be add nullptr as children?
```

Reply:
yeah, this is superfluous. I don’t think there is any situation where we want to add nullptr as children.